### ISSUE TITLE
Dash fails after pressing pause

### ISSUE THREAD

**Issue by Tralapo** [videojs/videojs-contrib-dash#10]
I'm having trouble with restarting a dash-livestream with Video.JS. Starting a stream is fine, but after you press 'pause', it's not possible to start the stream again. It results in the following errors if I press play again:

`dash.all.js:9 <video> play`
`dash.all.js:9 Refresh manifest in 5.402 seconds.`
`dash.all.js:9 [video] start`
`dash.all.js:9 <video> playing`
`dash.all.js:9 [video] Stalling Buffer`
`dash.all.js:9 [video] Waiting for more buffer before starting playback.`
`dash.all.js:9 <video> play`
`dash.all.js:9 Refresh manifest in 5.402 seconds.`
`dash.all.js:9 [video] start`
`113fc84e-d5b3-4555-895f-579b69f28cfd:1 GET blob:http%3A//mysite.ltd/113fc84e-d5b3-4555-895f-579b69f28cfd 404 (Not Found)`
`video.js:24 VIDEOJS: ERROR: (CODE:4 MEDIA_ERR_SRC_NOT_SUPPORTED) The video could not be loaded, either because the server or network failed or because the format is not supported. t.J {code: 4, message: "The video could not be loaded, either because the …rk failed or because the format is not supported."}F @ video.js:24t.log.error @ video.js:24s.error @ video.js:80s.Td @ video.js:125e @ video.js:14t.b.e.ba.e.ba @ video.js:7`
`dash.all.js:9 Video Element Error: MEDIA_ERR_SRC_NOT_SUPPORTED`
`dash.all.js:9 undefined`
`dash.all.js:9 [video] stop`

As you can see, I get a 404-error from some kind of 'blob' URL. This URL, here called mysite.ltd is NOT the URL of my stream, but the URL of the domain the player is located.

Why does the player call a URL like that and is this fixable? It should just call the manifest again I would say.

**Comment by Tralapo** [comment]
Interesting point: Microsoft Edge seems to be the only browser where it does work?

**Comment by Tralapo** [comment]
I can confirm this pause/play system does work with dash.js only. The stream will restart without problems then.

So the problem is caused by video.js somewhere.

**Comment by imbcmdth** [comment]
Do you have any examples of players with this problem? The blob urls are expected (that is how media-source extensions load sources.) Why video.js is trying to load a blob-url is the real question!

Any additional info would help me in figuring this out.

Thanks.

**Comment by Tralapo** [comment]
I found out what the problem is, but I don't know what to do now. Point of failure is [my VideoJS issue 1920](https://github.com/videojs/video.js/issues/1920). I use the following script to renew the HLS and RTMP version of the stream when pressing play after pause. This is because of the fact those aren't time based and when pressing play after one hour, they would be just looking at an hour old content, for example.

```javascript
    var myPlayer = videojs('my_video_1', {}, function(){
      var player = this;
      player.on("pause", function () {
        player.one("play", function () {
          player.load();
          player.play();
        });
      });
    });
```

When creating a example for you, I found out the problem with Dash is fixed when I delete that piece of code. It's indeed not needed for Dash, because it's time-based and it will pick up recent content after a few seconds.

But I do still use HLS and even RTMP as a fallback. HLS for Android (which fails with dash in my case) and RTMP with Flash for Firefox, which lacks support for HLS ánd Dash at the moment. I also had to put HLS first in row (see #6), to force Android to pick that one. Now Microsoft Edge supports HLS, so that browsers picks HLS to.

So, to be short: I do need the above javascript code to serve an always live view to HLS and RTMP users. Is there a way to fix the problem it creates for Dash? Or can you think of another way of doing this.

Live examples:

Player with javascript code, breaks Dash after pause/play, but HLS/RTMP are refreshed:
http://bit.ly/1N3kH8o

Player without javascript code. Now, Dash can restart. But HLS/RTMP will get out of sync after restart, which is what I want:
http://bit.ly/1P6Citw
